src/index.html

```diff
@@ -3,10 +3,357 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="container">
+      <header class="header">
+        <a href="#" class="logo">
+          <img
+             src="./images/logo.png"
+             alt="Moyo logo"
+             class="logo__image"
+           >
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link is-active">
+                Apple
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link"
+               data-qa="nav-hover"> Laptops & computers </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Gadgets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Tablets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Photo</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+
+      <main>
+        <div class="cards">
+          <div class="card" data-qa="card">
+            <img src="./images/imac.jpeg" alt="iMac"
+                class="card__img">
```

Comment:
The same as above, need to check and fix code style everywhere
```suggestion
            <img       
              src="./images/imac.jpeg"
              alt="iMac"
              class="card__img"
            >
```